### PR TITLE
REP-4465 Replace Enums with Constants

### PR DESCRIPTION
**Pull request by wdschei**
... and some other miscellaneous clean up as it was found.

### CODE REVIEW

**Comment by adrianjgeorge** [comment]
Lot of test failures

**Comment by wdschei** [comment]
Yep. It pointed out some brittle case sensitive header searches.
Fixing now.

File: repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/IdentityStatus.java [hunk 1]
```diff
@@ -22,19 +22,12 @@
 /**
  * @author fran
  */
-public enum IdentityStatus {
+public class IdentityStatus {
```
Comment (adrianjgeorge) [inline repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/IdentityStatus.java]: Depending on usage, this one might have actually made sense as an enum.

File: repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/PowerApiHeader.java [hunk 2]
```diff
@@ -22,28 +22,30 @@
...
-    NEXT_ROUTE("X-PP-Next-Route"),
-    USER("X-PP-User"),
-    GROUPS("X-PP-Groups"),
-    DOMAIN("X-Domain"),
-    X_CATALOG("x-catalog");
+    public static final String NEXT_ROUTE = "X-PP-Next-Route";
+    public static final String USER = "X-PP-User";
+    public static final String GROUPS = "X-PP-Groups";
+    public static final String DOMAIN = "X-Domain";
+    public static final String X_CATALOG = "x-catalog";
```
Comment (adrianjgeorge) [inline repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/PowerApiHeader.java]: This is a duplicate

File: repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/logging/apache/format/converters/DateConversionFormat.java [hunk 3]
```diff
@@ -19,27 +19,21 @@
  */
 package org.openrepose.commons.utils.logging.apache.format.converters;
 
-public enum DateConversionFormat {
+public class DateConversionFormat {
```
Comment (adrianjgeorge) [inline repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/logging/apache/format/converters/DateConversionFormat.java]: Not sure how i feel about this one. It doesn't quite feel right in it's new form.

File: repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/IdentityStatus.java (hunk 1 above)
Comment (wdschei) [inline repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/IdentityStatus.java]: I agree. This one could have gone either way. It just seems kind of heavyweight as an `enum` when it is just being used as `String`'s.

File: repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/logging/apache/format/converters/DateConversionFormat.java (hunk 3 above)
Comment (wdschei) [inline repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/logging/apache/format/converters/DateConversionFormat.java]: I think this class could be re-worked in the future, but this is the pattern used elsewhere. Again, this just seemed heavyweight in the original.

File: repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/PowerApiHeader.java (hunk 2 above)
Comment (wdschei) [inline repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/PowerApiHeader.java]: Removed.

**Pull request merged**